src/components/coin-icon/ChainBadge.js

```diff
@@ -65,8 +91,14 @@ export default function ChainBadge({
       badgeXPosition={badgeXPosition}
       badgeYPosition={badgeYPosition}
       iconSize={iconSize}
+      marginBottom={marginBottom}
```

Comment:
this is null unless a value is passed and is causing a crash, not sure what you want the default margin to be